ecr/templates/file_format_policy.json.tpl

```diff
@@ -0,0 +1,20 @@
+{
+  "Version": "2008-10-17",
+  "Statement": [
+    {
+      "Sid": "AllowPushPull",
```

Comment:
Is this policy just for pulling images rather than for pushing and pulling? Or have I misunderstood something?

Reply:
No you're right, I've changed the Sid to AllowPull